changelog/unreleased/issue-4547

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add support for `--json` option to `version` command
+
+Restic now supports outputting restic version and used go version and platform
+target via json when using the version command.
```

Comment:
Perhaps also change "go version and platform target" to "go version, OS and architecture" to match the other requested change.